client/asset/dcr/dcr.go

```diff
@@ -1596,13 +1596,76 @@ func (dcr *ExchangeWallet) AuditContract(coinID, contract, txData dex.Bytes) (*a
 	if txOut == nil {
 		return nil, asset.CoinNotFoundError
 	}
+
+	// Since txOut.Version with dcrd 1.6 is the transaction version NOT the
+	// output script version, we need to pull the entire transaction to get it.
+	// Unfortunately, for confirmed transactions we cannot do that with
+	// getrawtransaction without txindex. If confirmed, use
+	// GetTxOutResult.Confirmations and BestBlock to pull the block that is
+	// expected to include this tx.
```

Comment:
So unfortunate, but there's no other way to get the output script's version, and we really need it.